.github/workflows/docs.yml

```diff
@@ -17,42 +17,44 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      
+
       - uses: actions/setup-python@v4
         with:
           python-version: '3.x'
 
       - name: Setup doc dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r docs/requirements.txt
-      
+          python -m pip install -r docs/requirements.txt
+          python -m pip install .
+
       - name: Build docs
         run: |
           cd docs
           make html SPHINXOPTS="-W --keep-going -n"
+
       - name: Upload documentation
         uses: actions/upload-artifact@v4
         with:
           name: docs-html
           path: docs/_build/html/
-  
+
   apidocs:
     if: github.event.pull_request.merged != true || github.ref != 'refs/heads/main'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      
+
       - uses: actions/setup-python@v4
         with:
           python-version: '3.x'
 
       - name: Setup doc dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r apidocs/requirements.txt
-          pip install -e .
-      
+          python -m pip install -r apidocs/requirements.txt
+          python -m pip install .
```

Comment:
No need to have this be editable, so install it like normal.

All of this should be `pixi`fied, but that can be done in another PR to keep this as atomic as possible.